javascript-light-bulb.tar/javascript-light-bulb/main.js

```diff
@@ -0,0 +1,6 @@
+function changeStyle(){
+  const element=document.getElementById("container");
+  const element2=document.getElementById("button");
+  element.style.backgroundColor="beige";
```

Comment:
instead of using `element.style.backgroundColor` you could define classes in your css stylesheet with all of the colors set. Then your function could just be responsible to change the class(es) of the element.